template/src/project_name/main/settings.py

```diff
@@ -144,3 +146,22 @@
         },
     },
 }
+
+# Sentry
+sentry_sdk.init(
+    dsn=env("SENTRY_DSN", default=None),
+    environment=env("SENTRY_ENV", default="production"),
+    release=f"{{ project_name }}@{GIT_COMMIT_HASH}",
```

Comment:
@a-musing-moose @G-Rath I thought setting `release` version is also useful here.

Reply:
Good idea, though I'd consider omitting the project name as it'll always be the same and already expressed elsewhere (such as in the Sentry project itself), so having it just means it's more likely the unique part of the release identifier could get truncated and thus need a few extra clicks to get